static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -0,0 +1,21 @@
+# How to update your pipeline by editing dvc files
+
+> 1)using `dvc move` to edit file paths of outputs or dependencies.
```

Comment:
some introduction is required above. I don't quite understand what's happening here. Why it's `>`-ed. What is it about. Etc.

Reply:
hey, @Naba7  i will suggest that in introduction first describe why we need to update DVC-files?

for example: in `dvc run` we forget to mention any output file.

Reply:
okay. working on it